### PR TITLE
add python version for seq2seq

### PR DESCRIPTION
**Pull request by dding3**

### CODE REVIEW

**Comment by dding3** [comment]
Jekins passed. http://10.239.47.211:18888/job/ZOO-PR-Validation/1642/

File: pyzoo/zoo/models/seq2seq/seq2seq.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    @classmethod
+    def initialize(cls, bridge_type, decoder_hidden_size):
+        """
+        bridge_type: currently only support "dense | densenonlinear"
+        decoder_hiddenSize: hidden size of decoder
+        """
+        return Bridge(bridge_type, decoder_hidden_size, None)
+
+    @classmethod
+    def initialize_from_keraslayer(cls, bridge):
```
Comment (hkvision) [inline pyzoo/zoo/models/seq2seq/seq2seq.py]: keras_layer?

File: pyzoo/zoo/models/seq2seq/seq2seq.py [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+        rnns = createRNN(rnn_type.lower(), nlayers, hidden_size)
+        return RNNDecoder(rnns, embedding, input_shape)
+
+
+class Bridge(ZooKerasLayer):
+    """
+    defines how to transform encoder to decoder
+
+    # Arguments
+    bridge: keras layers used to do the transformation
```
Comment (hkvision) [inline pyzoo/zoo/models/seq2seq/seq2seq.py]: are these arguments doc out of date?

File: pyzoo/zoo/models/seq2seq/seq2seq.py [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    """
+    def __init__(self, rnns, embedding=None, input_shape=None):
+        super(RNNEncoder, self).__init__(None,
+                                         rnns,
+                                         embedding,
+                                         list(input_shape) if input_shape else None)
+
+    @classmethod
+    def initialize(cls, rnn_type, nlayers, hidden_size, embedding=None, input_shape=None):
+        rnns = createRNN(rnn_type.lower(), nlayers, hidden_size)
```
Comment (hkvision) [inline pyzoo/zoo/models/seq2seq/seq2seq.py]: put .lower() in createRNN?

**Comment by dding3** [comment]
Thanks for your review. Jekins pass. http://172.168.2.101:8080/view/ZOO-PR/job/ZOO-PR-Validation/1646/

**Pull request merged**